kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/sidePanels/SectionSidePanel/SectionEditor.vue

```diff
@@ -458,10 +452,10 @@
     left: 0;
     display: flex;
     justify-content: space-between;
-    padding: 1em;
+    padding: 1em 2em;
     margin-top: 1em;
     background-color: #ffffff;
-    border-top: 1px solid black;
+    @extend %dropshadow-1dp;
```

Comment:
Lets set a 2dp shadow to match the dropshadow we use inside the side panel component: 
https://github.com/allanoxdi/kolibri/blob/6ccb6b8772e399fbe8aa341c784954d7d21fe980/packages/kolibri-common/components/SidePanelModal/index.vue#L272